pkg/storaged/fsys-tab.jsx

```diff
@@ -253,12 +257,14 @@ export function mounting_dialog(client, block, mode, forced_options) {
                                                     if (mounted_at == new_dir)
                                                         return;
                                                     return (undo()
-                                                            .then(() => client.mount_at(block, old_dir))
-                                                            .then(() => Promise.reject(error))
+                                                            .then(() => {
+                                                                if (is_filesystem_mounted)
+                                                                    return client.mount_at(block, old_dir);
+                                                            })
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18180-20230117-101932-3c5e4716-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)